server/storage/types.go

```diff
@@ -139,6 +139,7 @@ type Cluster struct {
 	MinNodeCount      uint32
 	ReplicationFactor uint32
 	ShardTotal        uint32
+	PartitionTableNum uint32
```

Comment:
Need to modify Cluster pb in `ceresdbproto`.

Reply:
Update pb definition of `Cluster` in this pr: https://github.com/CeresDB/ceresdbproto/pull/39/files

Reply:
Let's remove it from cluster. It should not be a cluster option.